.travis.yml

```diff
@@ -1,28 +1,22 @@
-language: go
 sudo: required
-go:
-  - "1.9.x"
-  - "1.10.x"
-  - tip
+
+services:
+  - docker
 
 matrix:
-  allow_failures:
-    - go: tip
   fast_finish: true
 
 # Don't send out emails
 notifications:
   email: false
-
     
 before_script:
-  - wget http://mirrors.kernel.org/ubuntu/pool/universe/o/ostree/libostree-dev_2018.4-2_amd64.deb
-  - sudo dpkg -i --force-all ./libostree-dev_2018.4-2_amd64.deb
-  - mkdir -p $HOME/gopath/src/github.com/ostreedev/ostree-go/
-  - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/ostreedev/ostree-go/
+  - docker pull registry.fedoraproject.org/fedora:latest
+  - docker run -e GOPATH=/opt --cidfile=/tmp/cidfile registry.fedoraproject.org/fedora:latest /bin/bash -c "dnf install -y make golang go-compilers-golang-compiler findutils ostree ostree-devel glib-devel glib"
```

Comment:
You want `glib2`, not `glib` (though neither should be explicitly necessary anyways, ostree will pull it in via deps).